Bio/SeqFeature.py

```diff
@@ -1537,15 +1562,17 @@ def position(self):
 
     @property
     def extension(self):
-        """Legacy attribute to get extension (from left to right) as an integer (OBSOLETE)."""
+        """Legacy attribute to get the dimension (from left to right) as an integer (OBSOLETE)."""
```

Comment:
We're not actually enforcing that bit of flake8's checks yet are we? Right now the automated tests should be ignoring ``D402	First line should not be the function’s "signature"``

https://github.com/biopython/biopython/blob/master/Bio/.flake8

(This may be a false positive worth reporting to ``flake8``)